bin/now-scale.js

```diff
@@ -166,6 +166,10 @@ async function run({ token, config: { currentTeam } }) {
     error(`Could not find any deployments matching ${id}`)
     return process.exit(1)
   }
+  if (match.type === 'STATIC') {
+    console.log(`Could not scale static deployment: ${id}`)
```

Comment:
It's not being used elsewhere in these files either... so let's migrate the file at once eventually